node/pkg/driver/device_connectivity/device_connectivity_helper_scsigeneric.go

```diff
@@ -326,109 +251,25 @@ type OsDeviceConnectivityHelperGeneric struct {
 func NewOsDeviceConnectivityHelperGeneric(executer executer.ExecuterInterface) OsDeviceConnectivityHelperInterface {
 	return &OsDeviceConnectivityHelperGeneric{executer: executer}
 }
+func (o OsDeviceConnectivityHelperGeneric) WaitForDmToExist(volumeUuid string, maxRetries int, intervalSeconds int) (string, error) {
 
-func (o OsDeviceConnectivityHelperGeneric) WaitForPathToExist(devicePath string, maxRetries int, intervalSeconds int) ([]string, bool, error) {
-	/*
-				Description:
-					Try to find all the files
-					iSCSI -> /dev/disk/by-path/ip*-iscsi-<Array-WWN>-lun-<LUN-ID>
-		            FC   -> /dev/disk/by-path/pci-*-fc-<Array-WWN>-lun-<LUN-ID>
-					If not find then try again maxRetries.
-	*/
-
+	args := []string{"show", "maps", "raw", "format", "\"", "%d" + DmSepKye + "%w", "\""}
```

Comment:
use something like `wildcards := strings.Join([]string{"%d", "%w}, SEP)`

Reply:
why?

Reply:
`Join` is the opposite of `Split`, and is the appropriate function to use with separators.
and in general, string concatenation usually has worse performance and readability.

Reply:
where do you see string concatenation here?

Reply:
`"%d" + DmSepKye + "%w"`

Reply:
done